content/en/getting-started/installing.md

```diff
@@ -488,7 +488,15 @@ Hugo installed via Snap can write only inside the user’s `$HOME` directory---a
 
 What this installs depends on your Debian/Ubuntu version. On Ubuntu bionic (18.04), this installs the non-extended version without Sass/SCSS support. On Ubuntu disco (19.04), this installs the extended version with Sass/SCSS support.
 
-This option is not recommended because the Hugo in Linux package managers for Debian and Ubuntu is usually a few versions behind as described [here](https://github.com/gcushen/hugo-academic/issues/703)
+This option is not recommended because the Hugo in Linux package managers for Debian and Ubuntu is usually a few versions behind as described [here](https://github.com/gcushen/hugo-academic/issues/703).
+
+### MPR Package
+
+Hugo can be installed from the [makedeb Package Repository](https://mpr.makedeb.org/packages/hugo), which provides an up-to-date version of Hugo for users on Debian and Ubuntu based systems. To install it you need to first have [makedeb](https://makedeb.org/) installed, after which it can be installed via the `makedeb` command:
+
+    git clone 'https://mpr.makedeb.org/hugo.git'
+    cd hugo/
+    makedeb -si
```

Comment:
If you guys decide that you want to merge this, I should also note that makedeb doesn't have update capabilities itself outside of running `git pull` for the MPR package and then rebuilding.

There are some [MPR helpers](https://docs.makedeb.org/mpr/list-of-mpr-helpers/) that do support auto-updating, though I wasn't sure if I should include that or not.